index.html

```diff
@@ -164,117 +178,125 @@ <h2 class="visually-hidden">Популярные товары</h2>
-                <h2>О нас</h2>
-                <p>Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</p>
-                <p>Мы можем доставить ваш товар в самые отдаленные точки России! DEVICE работает со многими транспортными компаниями</p>
-                <ul class="dilivery_list">
-                    <li>Деловые линии</li>
-                    <li>Автотрейдинг</li>
-                    <li>ЖелДорЭкспедиция</li>
-                </ul>
-                <a href="#">Подробнее о нас</a>
-            </section>
-            <section class="contacts">
-                <h2>Контакты</h2>
-                <p>Вы можете забрать товар сами, заехав в наш офис. Заодно, вы сможете проверить работоспособность покупки. Всякое бывает.</p>                
-                <div>
-                    <img src=".//img/map.jpg" width="560" height="222" alt="Наш магазин расположен по адресу г. Москва, ул. Строителей, д.15">
+                <div class="logotypes">
+                    <ul class="logotypes-list">
+                        <li><a class="logotypes_link" href="#"><img src=".//img/content-img/logo-1.png" alt=""></a></li>
+                        <li><a class="logotypes_link" href="#"><img src=".//img/content-img/logo-2.png" alt=""></a></li>
+                        <li><a class="logotypes_link" href="#"><img src=".//img/content-img/logo-3.png" alt=""></a></li>
+                        <li><a class="logotypes_link" href="#"><img src=".//img/content-img/logo-4.png" alt=""></a></li>
+                    </ul>
                 </div>
-<!-- смотри раздел 4.3 specification-device, доделай, когда изучишь соответствующую тему  -->
-                <button type="button">Напишите наш</button>
-                <div class="modalwindow">
-                    <button type="button"><img src=".//img/modal-close.svg"></button>
-                    <form class="write-us" method="post" action="#">
-                        <div class="div_user-name">
-                            <label for="user-name">Ваше имя:</label>
-                            <input id="user-name" type="text" name="user-name" placeholder="Имя Фамилия">
-                        </div>
-                        <div class="div_email">
-                            <label for="email">Ваш e-mail:</label>
-                            <input id="email" type="email" name="user-email" placeholder="email@example.com">
-                        </div>
-                        <div class="div_text">
-                            <label for="text">Текст письма:</label>
-                            <textarea id="text" name="user-text" placeholder="В свободной форме"></textarea>
-                        </div>
-                        <button type="submit">Отправить</button>                            
-                    </form>
-                </div>                
-            </section>                    
+            </section>    
+            <section class="section-info">
```

Comment:
all section must have headers, at least hidden. think of just div wrapper for these two sections

Reply:
if all <section> must have <h> header then I missed at least two: section class="section-popular" and section class="services" Here have changed section to div. What about these two? Do add <h>?